plugins/org.yakindu.sct.commons/src/org/yakindu/sct/commons/EMFHelper.java

```diff
@@ -0,0 +1,35 @@
+package org.yakindu.sct.commons;
```

Comment:
can't you just use 
EClass clazz;
clazz.isSuperTypeOf(someOtherClass)
And delete EMFHelper

Reply:
Nope - I want to know if `B extends A` and `C extends A` are related, but they are not super types of each other.